recipes/sphinx-codeautolink/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "sphinx-codeautolink" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sphinx-codeautolink-{{ version }}.tar.gz
+  sha256: 9eabd4dfa0cf4dcb94020506f9ea1ede0f952a54e2704b70573a643c0f327aac
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - sphinx >=3.2.0
+    - beautifulsoup4
+    - dataclasses
```

Comment:
I don't suppose there's a way to make Python version dependent requirements? If not, go for 3.7+ and drop dataclasses IMO 😄

Reply:
From what I recall reading, `dataclasses` will only be explicitly added to Python3.6 builds. It's in the standard library for 3.7+, so I imagine that once conda-forge drops Python3.6 support completely, this won't be needed anymore.